packages/@n8n_io/eslint-config/shared.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @type {(dir: string, mode: 'frontend' | undefined) => import('@types/eslint').ESLint.ConfigData}
+ */
+exports.sharedOptions = (tsconfigRootDir, mode) => {
+	const isFrontend = mode === 'frontend';
+	const parser = isFrontend ? 'vue-eslint-parser' : '@typescript-eslint/parser';
+	const extraParserOptions = isFrontend
+		? {
+				extraFileExtensions: ['.vue'],
+				parser: {
+					ts: '@typescript-eslint/parser',
+					js: '@typescript-eslint/parser',
+					vue: 'vue-eslint-parser',
+					template: 'vue-eslint-parser',
+				},
+		  }
+		: {};
+
+	const settings = {
+		'import/parsers': {
+			'@typescript-eslint/parser': isFrontend ? ['.ts', '.vue'] : ['.ts'],
```

Comment:
looks like adding `.vue` here is adding unnecessary noise, as the linter is now trying to lint the ts code first, and then the entire vue file as typescript again, causing warnings like 
> `parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored